src/libraw.mk

```diff
@@ -0,0 +1,32 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := libraw
+$(PKG)_WEBSITE  := https://libraw.org
+$(PKG)_DESCR    := A library for reading RAW files obtained from digital photo cameras
+$(PKG)_VERSION  := 0.19.5
+$(PKG)_CHECKSUM := 40a262d7cc71702711a0faec106118ee004f86c86cc228281d12d16da03e02f5
+$(PKG)_SUBDIR   := LibRaw-$($(PKG)_VERSION)
+$(PKG)_FILE     := LibRaw-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://www.libraw.org/data/$($(PKG)_FILE)
+$(PKG)_DEPS     := cc jpeg lcms
+
+define $(PKG)_UPDATE
+    echo 'TODO: write update script for $(PKG).' >&2;
+    echo $($(PKG)_VERSION)
+endef
+
+define $(PKG)_BUILD
+    cd '$(1)' && ./configure \
+        $(MXE_CONFIGURE_OPTS) \
+        --disable-jasper \
+        --enable-jpeg \
+        --enable-lcms \
+        --disable-examples \
+        CXXFLAGS='-DLIBRAW_BUILDLIB' \
```

Comment:
Is this required for both static and shared builds?

Reply:
I think so:
```
#ifdef LIBRAW_BUILDLIB
#define DllDef __declspec(dllexport)
#else
#define DllDef __declspec(dllimport)
#endif
```
if we don't define it, then the APIs are marked with `__declspec(dllimport)`, which is wrong.

Reply:
Static build and linking wants no __declspec, you usually need something like https://github.com/mxe/mxe/issues/1647#issuecomment-275967915.

Reply:
Luckily it was easier that that, libraw already had a some preprocessor macros for that. Now I've updated the recipe, and verified that it builds both with static and dynamic linking.

There's one problem left, though: for the static build, I had to manually add `-lws2_32`. I'll check if I can find a better solution.

Reply:
Related issue: https://github.com/LibRaw/LibRaw/issues/267

I've fixed it with a patch. I cannot submit it upstream because it would break the Linux and macOS builds of libraw.

Reply:
Manually adding libs is fine, otherwise you need to patch configure.ac upstream.